target/linux/kirkwood/files-5.4/arch/arm/boot/dts/kirkwood-stora.dts

```diff
@@ -0,0 +1,227 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Can this file be renamed for consistency with existing kirkwood-netgear files?  The Netgear ReadyNAS Duo v2 (kirkwood-netgear_readynas_duo_v2.dts) and Ready NAS NV+ v2 (kirkwood-netgear_readynas_nv+_v2.dts) would indicate that kirkwood-netgear_stora.dts would be a more consistent name.

Reply:
The kernel already has the kirkwood-linksys-viper.dtsi while we have the kirkwood-e4200-v2 / kirkwood-ea3500 / kirkwood-ea4500 DTS files. I don't see this as an odd-ball here for the missing vendor name, but don't mind changing it.

Reply:
Since kernel does not care about consistent names at all, I'd also not follow that example. _If_ netgear should be in the name, we should probably go for kirkwood-netgear-stora.dts.

Reply:
> The hwmon-tc654 module is not a target-specific driver, IMO it would be better to add it to `package/kernel/linux/modules/hwmon.mk` directly.

I've broken it out to the standard hwmon.mk then, and then the apm821xx target might want to consider using it in its nand profile.